testlib/src/anker_context.rs

```diff
@@ -182,6 +188,66 @@ impl Context {
         context
     }
 
+    // Start a new Anker context with 10 StSOL and 10 UST in the liquidity
+    // provider AMM and initialized token pool.
+    pub async fn new_with_initialized_token_pool() -> Context {
+        const LIQUIDITY_AMOUNT: u64 = 10_000_000_000;
+        let mut context = Context::new().await;
+        context
+            .token_pool_context
+            .provide_liquidity(
+                &mut context.solido_context,
+                StLamports(LIQUIDITY_AMOUNT),
+                MicroUst(LIQUIDITY_AMOUNT),
```

Comment:
Ah, that’s why the decimal point was off earlier. This is not 10 UST, it is 10,000 UST; UST only has 6 decimals. I think we should just inline the numbers
```rust
StLamports(10_000_000_000),
MicroUst(10_000_000),
```

Reply:
Let's make 10_000 UST, and change the swapped amount in the tests above.